### ISSUE TITLE
高速にストロークを描いていると途中が抜けることがある

### ISSUE THREAD

**Issue by arcatdmz** [arch-inc/fabricjs-psbrush#14]
https://arch-inc.slack.com/archives/GAN0BK0CA/p1618282562024700?thread_ts=1618253421.021800&cid=GAN0BK0CA

**Comment by arcatdmz** [comment]
ストローク自体は記録されている雰囲気がある。(undo, redo できるので action 自体は起きているが、登録されているパスが存在しない)

**Comment by beeirl** [comment]
Any updates?

**Comment by arcatdmz** [comment]
@beeirl Not really, we won't have enough time to work on this, at least until the end of this March,  I guess.
